jupyter_book/cli/main.py

```diff
@@ -2,47 +2,57 @@
 
 IMPORTANT: Top-level imports should be minimised here, to improve CLI responsiveness
 """
-from glob import glob
 import os
 import os.path as op
-from pathlib import Path
 import shutil as sh
 import subprocess
 import sys
+from glob import iglob
+from pathlib import Path
 from textwrap import dedent
 from typing import Tuple
 
 import click
 
-from ..utils import _message_box, _error
+from jupyter_book.utils import _error, _message_box
```

Comment:
is there a reason that for some things we do packae imports (`from jupyter_book.utils import`) and for others we do `.`-style importing (`from .pluggable import`)?

Reply:
No reason, I think package imports are better (more explicit)

Reply:
I'm happy with either, but feel like we should just pick one style and go with it

Reply:
pep8 essentially advocates for absolute (package) imports, unless modules are particularly deeply nested (which is not the case here), so thats the style I would go for